Vagrantfile

```diff
@@ -12,10 +12,11 @@ grep -Fq 'cd /vagrant' ~/.bashrc || echo 'cd /vagrant' >> ~/.bashrc
 
 # Install system packages
 sudo apt-get update --quiet
```

Comment:
@brousch Shouldn't that get into `make install_system_*dependencies`?

Also, to avoid having to run the update twice, you could make the rule `make install_system_db_dependencies` depend on `make install_dependencies` (maybe renaming it in the process).